en/deploy-tidb-binlog.md

```diff
@@ -8,6 +8,10 @@ aliases: ['/docs/tidb-in-kubernetes/dev/deploy-tidb-binlog/']
 
 This document describes how to maintain [TiDB Binlog](https://docs.pingcap.com/tidb/stable/tidb-binlog-overview) of a TiDB cluster on Kubernetes.
 
+> **Warning:**
+>
+> Starting from TiDB v8.3.0, TiDB Binlog is deprecated, and is planned to be removed in a future release. Use [TiCDC](deploy-ticdc.md) instead for incremental data replication as an alternative.
```

Comment:
Similar to my above comment I would make the deprecation and removal process clearer. E.g. for this first sentence:

> Please note that TiDB Binlog replication was deprecated in v7.5.0, and TiDB Binlog will be fully deprecated starting from v8.3.0, with removal planned for a future release.